client/src/components/pages/[feed]/FeedHeader.tsx

```diff
@@ -159,6 +171,7 @@ const FeedHeader: React.FC<FeedHeaderProps> = (props): JSX.Element => {
 									<Statistic
 										suffix={item.suffix}
 										title={item.title}
+										// @ts-ignore
```

Comment:
Why need to apply @ts-ignore here? Production code should not allow this. Except for a reason.